core-services/prow/02_config/openshift/origin/_pluginconfig.yaml

```diff
@@ -8,6 +8,7 @@ label:
       - sttts
       - s-urbaniak
       - tkashem
+      - coreydaley
```

Comment:
/assign @coreydaley 

I am adding you to origin because we often make modifications to the extended build tests.